integration_tests/tests/s3-invoice-raw-store-tests.ts

```diff
@@ -35,12 +35,10 @@ describe("\n Unhappy path - Upload invalid pdf to the raw invoice bucket test\n"
         bucketName: rawInvoice.bucket,
         prefix: "failed",
       });
-      if (result.Contents === undefined) {
+      if (result === undefined) {
         return false;
       }
-      return result.Contents.some((items) =>
-        items.Key?.includes(rawInvoice.key)
-      );
+      return result.some((items) => items.key?.includes(rawInvoice.key));
```

Comment:
I think this might be a hangover from when we had the lint rule that made you do explicit null checks. You could one liner this into 
```
      return !!result && result.some((items) => items.key?.includes(rawInvoice.key));
```

Reply:
ah yes, changed to one liner